test/DebugInfo/LocalAddressSpace.ll

```diff
@@ -22,7 +22,7 @@
 ; CHECK: DW_TAG_variable
 ; CHECK-NEXT: DW_AT_name {{.*}} = "a")
 ; CHECK-NEXT: DW_AT_type {{.*}} "int")
-; CHECK-NEXT: DW_AT_decl_file {{.*}} ("/work/tmp{{[/\\]}}tmp.cl")
+; CHECK-NEXT: DW_AT_decl_file {{.*}} ("/work{{[/\\]}}tmp{{[/\\]}}tmp.cl")
```

Comment:
Maybe silly question, but should not we also check the first slash/backslash?
```suggestion
; CHECK-NEXT: DW_AT_decl_file {{.*}} ("{{[/\\]}}work{{[/\\]}}tmp{{[/\\]}}tmp.cl")
```

Reply:
Nope. This one remains to be the same as the dir name starts with /work in the test. See from the PR description:
`Note, the patch is not attempting to guess separators depending on the user's IR input, so for the linux <-> windows cross compilation the result will be odd (like it was before).`